src/renderer/components/dialogs/KeyTransfer.js

```diff
@@ -10,13 +10,27 @@ const {
 } = require('@blueprintjs/core')
 
 class KeyViewPanel extends React.Component {
+  formatAutocryptKey (autocryptKey) {
+    const splittedAutocryptKey = autocryptKey.split('-')
+    return splittedAutocryptKey.map((e, i) => {
+      const isLast = (i + 1) === splittedAutocryptKey.length
+      const isNThird = (i + 1) % 3 === 0
+
+      const addLineBreak = !isLast && isNThird
+      const addDash = !isLast
+
+      return e + (addDash ? ' - ' : '') + (addLineBreak ? '\n' : '')
+    }).join('')
+  }
```

Comment:
Can we make this a normal function? No reason to be a class method since it doesn't use any class state (i.e. not using `this`).

Reply:
I don't care but i like to have it in a class just to have it grouped with where it belongs to. Also i think i already saw similiar stuff.